Sandbox/DocumentationSnippets/FiltersApi/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "tutorial1",
```

Comment:
I think is worth to have a unique names, for this one it could be `filters-api-example`